mirrulations-core/src/mirrcore/attachment_saver.py

```diff
@@ -5,7 +5,7 @@ class AttachmentSaver():
     def __init__(self):
         pass
 
-    def save(self, data, path=os.path.join(os.getcwd(), 'mirrulations-core/tests/test_put_files')):
+    def save(self, data, path=os.path.join(os.getcwd())):
```

Comment:
The files are being successfully saved inside the docker container, but not to disk. We can docker exec into the work_server to see that they are being saved in there. Anyone know how to get on disk?